### PR TITLE
update copyright notice

### PR DESCRIPTION
**Pull request by cqc-melf**
# Description

Please summarise the changes.

# Related issues

Please mention any github issues addressed by this PR.

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented hard-to-understand parts of my code.
- [ ] I have made corresponding changes to the public API documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have updated the changelog with any user-facing changes.

### CODE REVIEW

File: tests/integration/backend_test.py [hunk 1]
```diff
@@ -1248,69 +1248,6 @@ def to_int(C: np.ndarray) -> int:
...
-            marks=pytest.mark.xfail(reason="https://github.com/CQCL/tket/issues/1174"),
-        ),
-    ],
-)
-@pytest.mark.timeout(120)
-def test_wasm_multivalue(
-    authenticated_quum_backend_qa: QuantinuumBackend, language: Language
-) -> None:
-    wasfile = WasmFileHandler(
-        str(Path(__file__).parent.parent / "wasm" / "multivalue.wasm")
```
Comment (cqc-alec) [inline tests/integration/backend_test.py:1270]: Can we delete this wasm file too (and the corresponding C file)?

Comment (cqc-melf) [reply tests/integration/backend_test.py:1270]: Yes, done in [f5a598e](https://github.com/CQCL/pytket-quantinuum/pull/418/commits/f5a598e3e4cee42c98e0670d4b84b3792f54dc80) and [9ed3e14](https://github.com/CQCL/pytket-quantinuum/pull/418/commits/9ed3e14f97c4faa28d8e91985876e3715766d039)

**Review by CalMacCQ — DISMISSED** [review]

File: tests/wasm/README.md [hunk 2]
```diff
@@ -9,14 +9,11 @@ at the top, and any other int-to-int functions below.
 Run:
 
 ```shell
-clang --target=wasm32 -mmultivalue -Xclang -target-abi -Xclang experimental-mv --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
+clang --target=wasm32 -mcollatz.c -Xclang -target-abi -Xclang experimental-mv --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
```
Comment (cqc-alec) [inline tests/wasm/README.md:12]:
```suggestion
clang --target=wasm32 -Xclang -target-abi -Xclang experimental-mv --no-standard-libraries -Wl,--export-all -Wl,--no-entry -o <filename>.wasm <filename>.c
```

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: tests/wasm/README.md (hunk 2 above)
Comment (cqc-melf) [reply tests/wasm/README.md]: I thought this was the filename as well... Thank you for spotting!

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**